_health-care/_js/components/ContinueButton.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+/**
+ * A component for the continue button to navigate through panels of questions.
+ */
+class ContinueButton extends React.Component {
+  render() {
+    return (
+      <div className="row progress-buttons">
+        <div className="small-8 columns">
+          <button
+              className="usa-button-primary"
+              id="continueButton"
```

Comment:
no `id`s that aren't gensymed with _.uniqueId() otherwise you will eventually clash in the DOM.